### ISSUE TITLE
Rename File Fails Silently

### ISSUE THREAD

**Issue by lynyus** [JabRef/jabref#3172]
Renaming a linked file fails if there already exists a file with the name.

In the General tab of the Entry Editor: Right click on the linked file -> Rename File. A popup that states the new file name appears. Confirming it results in a failure without any message. The other file is not overwritten and the operation apparently has no effect.

I would expect that a popup message appears and asks to user to confirm the overwrite.

JabRef 4.0-dev
Mac OS X 10.12.6 x86_64 
Java 1.8.0_144

**Comment by Siedlerchr** [comment]
Have a look at 
```
public class LinkedFileViewModel extends AbstractViewModel {

 public void rename() { 
```

**Comment by Andrew-Lev** [comment]
It sounds like adding a check for existing files in that directory with the targetFileName and then showing the error dialog would take care of this.  I can fix this fairly quickly, want me to work on it?

**Comment by Andrew-Lev** [comment]
Didnt take very long at all actually.  I'll make a pull request once someone confirms.

![samefilenameerror](https://user-images.githubusercontent.com/17808285/29958237-8db997f8-8ea6-11e7-85f5-1ba9a0d0a718.png)

**Comment by lynyus** [comment]
Yeah, cool stuff! Just open the PR, and we'll have a look at it!

**Comment by lynyus** [comment]
fixed in #3211. Thanks @Andrew-Lev!
